src/pages/iou/request/IOURequestStartPage.js

```diff
@@ -103,7 +104,7 @@ function IOURequestStartPage({
 
     const isExpenseChat = ReportUtils.isPolicyExpenseChat(report);
     const isExpenseReport = ReportUtils.isExpenseReport(report);
-    const shouldDisplayDistanceRequest = isExpenseChat || isExpenseReport || isFromGlobalCreate;
+    const shouldDisplayDistanceRequest = isExpenseChat || isExpenseReport || isFromGlobalCreate || iouType === CONST.IOU.TYPE.TRACK_EXPENSE;
```

Comment:
We will want to hide the distance tab in the track expense selector for now - for context it's in the process of being refactored to fit the IOU flow